src/libraries/pkg/runtime.native.System.IO.Ports/netcoreapp.rids.props

```diff
@@ -1,5 +1,5 @@
 ﻿<Project>
-  <ItemGroup>
+  <ItemGroup Condition="'$(PackAllLibraries)' == 'true'">
```

Comment:
Is this condition necessary?

Reply:
It is necessary. The runtime-specific packages were built without it.